source/regression1.md

```diff
@@ -596,23 +596,31 @@ on `sacr_gridsearch`. As we did in the {ref}`classification2` chapter,
 we will wrap the `cv_results_` output in a data frame, extract
 only the relevant columns, compute the standard error based on 5 folds, 
 and rename the parameter column to be more readable.
+In `scikit-learn`, it is easier to work with the input features as a data frame,
+rather than a series. So, when we select a single column,
+we pass the name of the column as a list rather than a string
+to return a data frame
+as we learned in the {ref}`wrangling` chapter.
+The target variable can be a series,
+so we can pass its column name as a string.
 
 ```{code-cell} ipython3
-# fit the GridSearchCV object 
+# fit the GridSearchCV object
 sacr_fit = sacr_gridsearch.fit(
-                  sacramento_train[["sqft"]],
-                  sacramento_train[["price"]]
-              )
-# retrieve the CV scores
-sacr_results = pd.DataFrame(sacr_fit.cv_results_)[
-    ["param_kneighborsregressor__n_neighbors", "mean_test_score", "std_test_score"]
-]
-sacr_results = sacr_results.assign(
-    sem_test_score = sacr_results["std_test_score"] / 5**(1/2)
-).rename(
-    columns = {"param_kneighborsregressor__n_neighbors" : "n_neighbors"}
-).drop(
-    columns = ["std_test_score"]
+    sacramento_train[["sqft"]],  # A single column data frame
+    sacramento_train["price"]  # A series
+)
+
+# Retrieve the CV scores
+sacr_results = (
+    pd.DataFrame(sacr_fit.cv_results_)[[
+        "param_kneighborsregressor__n_neighbors",
+        "mean_test_score",
+        "std_test_score"
+    ]]
+    .assign(sem_test_score=sacr_results["std_test_score"] / 5**(1/2))
```

Comment:
@joelostblom this generates an error in the textbook. `sacr_results` hasn't been created yet -- this is why it was done in two steps before. I will fix it.